@here/harp-mapview/lib/Utils.ts

```diff
@@ -1631,7 +1631,7 @@ export namespace MapViewUtils {
                 if (tile.tileLoader !== undefined && !tile.tileLoader.isFinished) {
```

Comment:
Put both conditions of tile.tileLoader and tileGeometryLoader into soemthing like tile.isLoading, no need for Utils then to know the internals

Reply:
there is also 2 references to tile.tileLoader in the VisibleTileSet which look like they could be using this too, something like if(tile.isLoading) {tile.canccel())

Reply:
this might lead to more loading tiles being found and canceled, if the check includes the tileGeometryLoading

Reply:
This code is wrong. If the tileLoader is not finished then tile.allGeometryLoaded is false as well. Just checking allGeometryLoaded should suffice.